.github/workflows/push.yml

```diff
@@ -480,3 +483,14 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build and Push fio image
         run: make fio
+  
+  litmus-pg-jmeter:
+    runs-on: ubuntu-latest
+    needs: changes
+    if: needs.changes.outputs.litmus-pg-jmeter == 'true'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Building litmus-pg-jmeter image
+        run: make _push_litmus_pg_jmeter
```

Comment:
you can also add [make docker.buildx](https://github.com/litmuschaos/test-tools/blob/7d8559fe8effb62c1c87aef8050a7c7df62f90e2/.github/workflows/push.yml#L188) before calling push